src/registry.js

```diff
@@ -107,6 +108,11 @@ module.exports = (token, LMSR, web3, address, defaultOptions) => {
     return listing
   }
 
+  const getListingHash = async (listingHash) => {
```

Comment:
maybe change `listingHash` to `listingTitle` ... I know it needs to be done in other spots too, but could start here